components/TopNav.vue

```diff
@@ -11,30 +11,8 @@
                 About Us
             </NuxtLink>
 
-            <div class="hidden cursor-pointer md:hidden">
-                <img @click="toggleMenu"
-                    src="../assets/images/hamburgerMenu.svg"
-                    alt="hamburger menu" />
-                <div v-if="showMenu">
-                    <div>
-                        <span class="block">Home</span>
-                        <span class="block">About</span>
-                    </div>
-                </div>
-            </div>
+            <HamburgerComponent />
```

Comment:
@Tch4lla I think a better name would be `HamburgerMenu`. Typically you don't need to include component in the name itself. We can update this in the next PR ✅